src/Pages/SurveyPage/style.ts

```diff
@@ -0,0 +1,177 @@
+import styled from "styled-components";
+
+export const MainContainer = styled.div`
+  background-color: black;
+  width: 100%;
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+`;
+
+export const SurveMainyBox = styled.div`
+  width: 800px;
+  height: 80%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 30px;
+`;
+
+export const SurveyMainTitle = styled.h1`
+  color: white;
+  font-size: 24px;
+  font-weight: 700;
+  text-align: center;
+  line-height: 30px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+`;
+
+export const SurveyMainItemBox = styled.div`
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  gap: 30px;
+`;
+
+export const SurveyMainItem = styled.img<{ selected: boolean }>`
+  cursor: pointer;
+  filter: ${(props) =>
+    props.selected ? "brightness(100%)" : "brightness(0.4)"};
```

Comment:
```suggestion
  filter: ${({selected}) =>
    selected ? "brightness(100%)" : "brightness(0.4)"};
```
구조 분해 문법을 사용하면 좋을거 같습니다